core/components/atoms/placeholderImage/__stories__/variants/Size.story.jsx

```diff
@@ -14,11 +14,12 @@ export const size = () => {
       {sizes.map((PlaceholderSize, ind) => {
         return (
           <div key={ind} className="mr-7">
-            <div style={{ height: '50px' }}>
+            <Text weight="strong">{PlaceholderSize.charAt(0).toUpperCase() + PlaceholderSize.slice(1)}</Text>
+            <br />
+            <br />
+            <div>
```

Comment:
Please remove these unnecessary `br` and `div` tag

Reply:
Those were used earlier as well and are used for right spacing